src/Compiler/Service/ServiceDeclarationLists.fs

```diff
@@ -430,7 +429,7 @@ module DeclarationListHelpers =
 
         | Item.AnonRecdField(anon, argTys, i, _) -> 
             let argTy = argTys[i]
-            let nm = anon.SortedNames[i]
+            let nm = anon.DisplayNameByIdx i
```

Comment:
This should fix quick info when hovering over `Thing with space` in 

```fsharp
type R = {| ``Thing with space``: int |}
```

Reply:
Yes. The ticket is [here](https://github.com/dotnet/fsharp/issues/13609), the test is [here](https://github.com/dotnet/fsharp/pull/13552/files#diff-1e6dc3c6f1f00f584bbc25dce0ef0259e996c14c91d1b58b58ace6e297325cabR506).